### PR TITLE
Ocm api load Job

### PR DESCRIPTION
**Pull request by chentex**
## Overview

Some workloads doesn't need OCP environemnt. For example, ocm-api-load which tests OCM server can be run as a standalone application on a jump host. Any tool which can be run on a jump host but needed to triggered using airflow can use this mechanism. Assumptions in this approach

* tool is run from the jump host
* user has to manually setup needed packages in the jump host
* user provided script for benchmark task should handle everything like triggering the tool on jump host, scrapping metrics, pushing data to snappy and cleanup.

> At no point this Job will ever create resources on any cloud provider

Passing build: [102](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/vzepedam-e2e-benchmarking-multibranch-pipeline/job/ocm-api-load/102/)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                    withCredentials([
+                            file(credentialsId: 'ocm-al-aws', variable: 'AWS_CREDS' ),
+                            file(credentialsId: 'ocm-al-infra', variable: 'INFRA' ),
+                            string(credentialsId: 'ocm-al-ocm-token', variable: 'OCM_TOKEN' ),
+                            string(credentialsId: 'ocm-al-prom-token', variable: 'PROM_TOKEN' ),
+                            string(credentialsId: 'ocm-al-server-password', variable: 'ES_SERVER_PASS' ),
+                            string(credentialsId: 'ocm-al-sshkey-token', variable: 'SSHKEY_TOKEN' ),
+                    ]) {
+                        env.AWS_ACCESS_KEY_ID = sh(script: "cat \$AWS_CREDS | awk -F' = ' '/^aws_access_key_id/ {print \$2}'", returnStdout: true).trim()
+                        env.AWS_SECRET_ACCESS_KEY = sh(script: "cat \$AWS_CREDS | awk -F' = ' '/^aws_secret_access_key/ {print \$2}'", returnStdout: true).trim()
```
Comment (paigerube14) [inline Jenkinsfile:161]: lets try to find a way to not print these parameters out to the console in a readable way

**Comment by paigerube14** [comment]
See error in the middle of the run

```
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `${which_declare}': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `)': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `|': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `/usr/bin/which': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--tty-only': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--read-alias': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--read-functions': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--show-tilde': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--show-dot': not a valid identifier
06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `$@=': not a valid identifier
06-26 01:54:07.979  + true
```

**Comment by chentex** [comment]
> See error in the middle of the run
> 
> ```
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `${which_declare}': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `)': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `|': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `/usr/bin/which': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--tty-only': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--read-alias': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--read-functions': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--show-tilde': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `--show-dot': not a valid identifier
> 06-26 01:54:07.979  /tmp/76270cb3-102-20240626/run_ocm_api_load.sh: line 50: export: `$@=': not a valid identifier
> 06-26 01:54:07.979  + true
> ```

Yes, this don't happen all the time, it seems to be only on some workers that have complex data in their environment variables, and when the jump host tries to process them to it's env it gives those errors, it is not affecting the functionality of the job.

File: Jenkinsfile (hunk 1 above)
Comment (chentex) [reply Jenkinsfile:161]: I removed the debug options on the script that should take care of it, thank you.

Comment (chentex) [reply Jenkinsfile:161]: Latest build no longer displeys the keys
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/vzepedam-e2e-benchmarking-multibranch-pipeline/job/ocm-api-load/104/consoleFull

**Pull request merged**